SYCL/AtomicRef/xor_local.cpp

```diff
@@ -3,145 +3,7 @@
 // separately. This is controlled by macros, defined by RUN commands. Defaults
 // (no macro for a group) are: 32 bit, relaxed and device.
```

Comment:
You are changing this behavior, so this comment can be removed (from all cpp files).